README.md

```diff
@@ -141,49 +177,41 @@ the snippet above with `<arg value="-F"/>`.
 
 To run diktat to check/fix code style - run `$ mvn antrun:run@diktat`.
 
-### Use the new diktat-maven-plugin
+## Run with Gradle
+### Use diktat-gradle-plugin
+You can see how the plugin is configured in our project for self-checks: [build.gradle.kts](build.gradle.kts).
+Add this plugin to your `build.gradle.kts`:
```

Comment:
any version for `build.gradle`?

Reply:
Yes, I have tested it in groovy, but I'd better add different example to `diktat-examples`.